### PR TITLE
Feat : Label edit

### PR DESCRIPTION
**Pull request by dikshadhote**
Following features included in PR:

1. Add not to labels
2. Distinguish different notes in UI according to labels 
image
3. Edit note in home page 
image

4. Edit not in archive page
image

### CODE REVIEW

**Review by cjinc121 — APPROVED** [review]
UI looks awesome 
code is clean
liked the api calls

File: src/ApiServices/ApiServices.js [hunk 1]
```diff
@@ -62,6 +62,27 @@ const getNotedata = async () => {
...
 };
 
+// Edit note data /notes/:noteId
+
+const editNoteData = async (note) => {
+  try {
+    const res = await axios.post(
+      `/api/notes/${note._id}`,
+      {
+        note: { ...note },
```
Comment (JantuDeb) [inline src/ApiServices/ApiServices.js]: You can directly add `note` here

File: src/ApiServices/ApiServices.js [hunk 2]
```diff
@@ -139,6 +160,28 @@ const moveToTrashFromArchive = async (note) => {
...
 };
 
+// Edit from Archive
+
+const editNoteFromArchive = async (note) => {
+  try {
+    const res = await axios.post(
+      `/api/archives/${note._id}`,
+      {
+        note: { ...note },
```
Comment (JantuDeb) [inline src/ApiServices/ApiServices.js]: Same here

**Review by JantuDeb — APPROVED** [review]
Overall the code looks fine.

**Pull request merged**